lib/matplotlib/contour.py

```diff
@@ -964,12 +964,30 @@ def changed(self):
             label.set_color(self.labelMappable.to_rgba(cv))
         super().changed()
 
-    def _autolev(self, N):
+    def _ensure_locator_exists(self, N):
         """
-        Select contour levels to span the data.
+        Set a locator on this ContourSet if it's not already set.
+
+        If *N* is an int, it is used as the target number of levels.
+        Otherwise when *N* is None, a reasonable default is chosen;
+        for logscales the LogLocator chooses, N=7 is the default
+        otherwise.
 
-        The target number of levels, *N*, is used only when the
-        scale is not log and default locator is used.
+        Parameters
+        ----------
+        N : int or None
```

Comment:
This really should go in the description of `N`, no?
```suggestion
        Parameters
        ----------
        N : int or None
            If *N* is an int, it is used as the target number of levels.
            Otherwise when *N* is None, a reasonable default is chosen;
            for log scales the LogLocator chooses, N=7 is the default
            otherwise.
```